pkg/cmd/admin/admin.go

```diff
@@ -22,7 +22,8 @@ import (
 	"github.com/openshift/origin/pkg/version"
 )
 
-const adminLong = `OpenShift Administrative Commands
+const adminLong = `
+OpenShift Administrative Commands
```

Comment:
Commander trims automatically.  This is just making our use consistent -
the constants are more readable this way.

On Mon, Jul 13, 2015 at 7:43 AM, David Eads <notifications@github.com>
wrote:

> In pkg/cmd/admin/admin.go
> <https://github.com/openshift/origin/pull/3669#discussion_r34454942>:
>
> > @@ -22,7 +22,8 @@ import (
> >  	"github.com/openshift/origin/pkg/version"
> >  )
> >
> > -const adminLong = `OpenShift Administrative Commands
> > +const adminLong = `
> > +OpenShift Administrative Commands
>
> Why add the newline that's just going to get trimmed later on? Also, why
> force someone making a template to know that they need to trim?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/3669/files#r34454942>.
>



-- 
Clayton Coleman | Lead Engineer, OpenShift